schematic/models/metadata.py

```diff
@@ -280,6 +280,7 @@ def submit_metadata_manifest(
         validate_component: str = None,
         use_schema_label: bool = True,
         hide_blanks: bool = False,
+        input_token: str = None
     ) -> bool:
```

Comment:
Change the return type after double checking other functions are not calling this function and expecting a bool return (probably not).